spec/console_spec.rb

```diff
@@ -0,0 +1,213 @@
+RSpec.describe Console do
+  let(:game) { Game.new }
+  let(:hint) { 'Hint' }
+  let(:test_guess) { '1234' }
+  let(:lose) { '----' }
+  let(:win) { '++++' }
+  let(:agree_command) { 'Yes' }
+  let(:disagree_command) { 'No' }
+  let(:name) { 'Rostik' }
+  let(:name_difficulties) { { difficulties: 'easy, medium, hell' } }
+  let(:easy) { 'Easy' }
+  let(:medium) { 'Medium' }
+  let(:hell) { 'Hell' }
```

Comment:
create let where you need it!
If some describes don't need some let - don't create it for all describes - just create it inside some of them where you need it